docker/dagmc-viz.dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM  ubuntu:18.04
+
+# set timezone
+ENV HOME /root
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# ubuntu setup
+RUN apt-get update
+RUN apt-get install -y --fix-missing \
+    software-properties-common \
+    wget \
+    build-essential \
+    python-setuptools \
+    python-numpy \
+    python-pip \
+    python-tables \
+    python-matplotlib \
+    git \
+    cmake \
+    vim \
+    emacs \
+    nano \
+    gfortran \
+    autoconf \
+    libtool \
+    cpio \
+    libblas-dev \
+    liblapack-dev \
+    libhdf5-dev \
+    libgl1-mesa-glx \
+    libgl1-mesa-dev \
+    libsm6 \
+    libxt6 \
+    libglu1-mesa \
+    libpcre3-dev \
+    libpython-dev
+RUN pip install Cython xmldiff pytest
```

Comment:
I would suggest to get all the python packages from the same source (`apt-get` or `pip`)